snippets/dart_snippets/lib/lnurl_withdraw.dart

```diff
@@ -3,21 +3,22 @@ import 'package:breez_sdk/bridge_generated.dart';
 
 Future<void> lnurlWithdraw() async {
   // ANCHOR: lnurl-withdraw
-  /// Endpoint can also be of the form:
-  /// lnurlp://domain.com/lnurl-pay?key=val
-  /// lnurl1dp68gurn8ghj7mr0vdskc6r0wd6z7mrww4excttsv9un7um9wdekjmmw84jxywf5x43rvv35xgmr2enrxanr2cfcvsmnwe3jxcukvde48qukgdec89snwde3vfjxvepjxpjnjvtpxd3kvdnxx5crxwpjvyunsephsz36jf
-  String lnurlPayUrl = "lightning@address.com";
+  // Endpoint can also be of the form:
+  // lnurlw://domain.com/lnurl-withdraw?key=val
```

Comment:
```suggestion
  /// Endpoint can also be of the form:
  /// lnurlw://domain.com/lnurl-withdraw?key=val
```

Reply:
Not needed anymore after ae4bc95fb64d34677b60e9ebe8e6039153b5f554